README-CN.md

```diff
@@ -8,20 +8,20 @@
 
 ## CITA 是什么？
 
-CITA（ Cryptape Inter-enterprise Trust Automation ）是一个面向企业级应用的支持智能合约的区块链框架，
+CITA（ Cryptape Inter-enterprise Trust Automation ）是一个面向企业级应用的支持智能合约的高性能区块链内核，
 旨在为企业级区块链应用提供一个稳固、高效、灵活、可适应未来的运行平台。CITA 将区块链节点的必要功能解耦为六个微服务：RPC，Auth，Consensus，Chain，Executor，Network。各组件之间通过消息总线交换信息相互协作。通过配置和定制相应的服务，CITA 能够满足企业级用户的全部需要。
 
 - **水平扩展性**
 
   在 CITA 的微服务架构中，“节点”是一个逻辑概念，有可能是一台服务器（一台服务器上面运行一组微服务），
   也有可能是一组服务器组成的集群，同时 CITA 还支持部署在云服务器上，充分利用了各种服务器硬件来提升处理能力。
-  节点与节点之间通过P2P通信，节点内部各模块通过消息总线通信，这一点与 Fabric 仅仅在共识模块运用消息总线通信完全不同。
+  节点与节点之间通过 P2P 通信，节点内部各模块通过消息总线通信，这一点与 Fabric 仅仅在共识模块运用消息总线通信完全不同。
```

Comment:
It's better to remove the `Fabric` info.
We do not need to emphasize this.

Reply:
Good advice.